src/api/app/assets/stylesheets/webui2/webui2.css.scss

```diff
@@ -16,7 +16,9 @@ $secondary: #39c;
 $link-color: $primary;
 $link-hover-color: $secondary;
 $headings-color: $primary;
-$body-bg: #f6f6f6;
+$gray-100: #f8f9fa;
+$body-bg: $gray-100;
+$gray-200: #e9ecef;
```

Comment:
why is this called `gray-200`? :thinking: I would have `gray-200` to be `C8C8C8`

Reply:
Because that is the name bootstrap uses

Reply:
but for C8C8C8... :confused:

Reply:
and do we need to define it again? It is already in the [`variables`](https://github.com/twbs/bootstrap/blob/v4-dev/scss/_variables.scss) file